web/docs/Backend Starters/Decoupled WordPress/local-development.md

```diff
@@ -0,0 +1,6 @@
+# Local Development Documentation
+
+## lando setup for [decoupled-wordpress-composer-managed](https://github.com/pantheon-systems/decoupled-wordpress-composer-managed)
```

Comment:
The acceptance criteria for this ticket outlines that we have a setup for backend only, and another for a combined front end and backend. Could you add a backend only section here? That may just be the standard process for running Lando init.